### PR TITLE
fix: groups UI in identity

### PR DESCRIPTION
**Pull request by matthewBearCamunda**
## Description

Viewing details of a group is still broke, but with this change you can...

- [x] Show the groups page without error
- [x] Create a group
- [x] Display list of groups
- [x] Edit group
- [x] Delete group
- [ ] View details of group

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

File: search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/ServiceTransformers.java [hunk 1]
```diff
@@ -227,7 +236,7 @@ public static void initializeTransformers(
     mappers.put(VariableSort.class, new VariableFieldSortingTransformer());
     mappers.put(AuthorizationSort.class, new AuthorizationFieldSortingTransformer());
     mappers.put(RoleSort.class, new RoleFieldSortingTransformer());
-    mappers.put(TenantSort.class, new TenantFieldSortingTransformer());
```
Comment (matthewBearCamunda) [inline search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/ServiceTransformers.java:230]: This was a duplicate addition that I removed. See line 232 for where it is added

**Review by Ben-Sheppard — APPROVED** [review]
Tested locally and functionally its all good :) - not sure if @maryarm or @emulbreh  wants to review the code also.

Also there are changes in here that are also in another PR from @koevskinikola I believe so may need some syncing.

As functionally its fine I'm going to approve

File: search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/sort/GroupFieldSortingTransformer.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.clients.transformers.sort;
+
+import static io.camunda.webapps.schema.descriptors.usermanagement.index.TenantIndex.KEY;
+import static io.camunda.webapps.schema.descriptors.usermanagement.index.TenantIndex.NAME;
```
Comment (koevskinikola) [inline search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/sort/GroupFieldSortingTransformer.java:11]: ❌ I guess we should use the `GroupIndex` properties instead.

**Comment by koevskinikola** [comment]
> Also there are changes in here that are also in another PR from @koevskinikola I believe so may need some syncing.

Yes, some of the code is added with [this PR](https://github.com/camunda/camunda/pull/25921). @matthewBearCamunda I'll refactor some additional properties and merge that PR, it would be good if you could wait on it.

**Comment by koevskinikola** [comment]
@matthewBearCamunda https://github.com/camunda/camunda/pull/25921 is merged, you can rebase now :)

**Comment by matthewBearCamunda** [comment]
Delete and Edit group both broke after the recent merge from `main`

I'll need to make a change to fix both of them

**Comment by matthewBearCamunda** [comment]
Edit functionality works again, still need to make changes for delete. It is encountering a zeebe error on delete

**Comment by matthewBearCamunda** [comment]
The issues with delete have been solved! This is ready for a a quick re-review @Ben-Sheppard

**Review by Ben-Sheppard — APPROVED** [review]
Seems reasonable to me, would be good to check with @koevskinikola as he had a comment before :)

**Comment by Ben-Sheppard** [comment]
I'm going to merge this and we can follow up in a targeted PR for the details view as there is also an endpoint needed for that

**Pull request merged**